source/contentRecog/recogUi.py

```diff
@@ -15,32 +15,94 @@
 import ui
 import screenBitmap
 import NVDAObjects.window
+from NVDAObjects.behaviors import LiveText
 import controlTypes
 import browseMode
 import cursorManager
 import eventHandler
 import textInfos
 from logHandler import log
 import queueHandler
+import core
 from . import RecogImageInfo
 
-class RecogResultNVDAObject(cursorManager.CursorManager, NVDAObjects.window.Window):
+
+class RecogResultNVDAObject(cursorManager.CursorManager, LiveText, NVDAObjects.window.Window):
 	"""Fake NVDAObject used to present a recognition result in a cursor manager.
 	This allows the user to read the result with cursor keys, etc.
 	Pressing enter will activate (e.g. click) the text at the cursor.
 	Pressing escape dismisses the recognition result.
 	"""
+	#: How often (in ms) to perform recognition.
+	REFRESH_INTERVAL = 1500

```

Comment:
Consider making this a config option?

Reply:
If we decide that this interval should be configurable I believe we should be able to set it individually per recognition provider.